### DISCUSSION TITLE
To receive email notifications for all Discussions activity.

### DISCUSSION THREAD

**Discussion by john-harrold** [nlmixr2/nlmixr2#27] [Show and tell]
To be notified when anyone posts to the discussion

- Select Watch --> Custom
- Check Discussions
- Click Apply

<img width="313" alt="image" src="https://user-images.githubusercontent.com/3170004/177648534-039eb4fb-931b-4d59-815a-5142454ea797.png">

**Comment by mattfidler** [comment]
Awesome @john-harrold 

I pinned this
